Lib/tkinter/__init__.py

```diff
@@ -934,7 +934,7 @@ def tk_busy_configure(self, cnf=None, **kw):
         tk_busy_hold().  Options may have any of the values accepted by
         tk_busy_hold().
 
-        Please note that the option database is referenced by the widget
+        Please note that the option database is11 referenced by the widget
```

Comment:
Fix typo:

```suggestion
        Please note that the option database is referenced by the widget
```

Reply:
Yes, I accidentally touched it. I just fixed it